vendor/assets/stylesheets/mailer.scss

```diff
@@ -0,0 +1,62 @@
+.mailer-content {
+  max-width: 900px;
+}
+
+.mailer-first-content {
+  background-color: #f7ede4;
+  max-width: 900px;
+}
+
+.message-title {
+  text-align: center;
+  font-weight: bolder;
+  padding-top: 30px;
+}
+
+.hr-line {
+  width: 90%;
+  text-align: center;
+}
+
+.basic-p {
+  margin: 20px;
+  max-width: 900px;
+}
+
+.paragraf {
+  width: 90%;
+  margin: 5px 5%;
+  font-size: 20px;
+}
+
+.link-btn {
+  background-color: mediumpurple;
+  border: none;
+  color: white;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+}
+
+.centered-paragraf {
+  text-align: center;
+}
+
+.last-paragraf {
+  padding-bottom: 20px;
+}
+
+.paragraf-image {
+  max-width: 900px;
```

Comment:
you have 900px in 4 places, please DRY it

Reply:
Ok, it's done!